packages/link-modules/src/definitions/product-shipping-profile.ts

```diff
@@ -0,0 +1,52 @@
+import { Modules } from "@medusajs/modules-sdk"
+import { ModuleJoinerConfig } from "@medusajs/types"
+import { LINKS } from "../links"
+
+export const ProductShippingProfile: ModuleJoinerConfig = {
+  serviceName: LINKS.ProductShippingProfile,
```

Comment:
thought(non-blocking): Would it make sense for this link to contain an extra field `type` which can be `default`, `gift_card`, or `custom` as with our model in the core?

Reply:
it is already part of the shipping_profile table, any reason to also have it in the pivot table?
@carlos-r-l-rodrigues since it is a pivot table, the aggregation will take care of the fact that the profiles result is a list and aggregate the first result to the shipping_profile alias?

Reply:
Yes, if it is not explicitly flagged with `isList` property it will transform it in an object if there is only 1 entry returned